api/src/public/login/bowser.js

```diff
@@ -0,0 +1 @@
+../../../../webapp/node_modules/bowser/bundled.js
```

Comment:
This is not a 1-line text file but a symbolic link pointing to a dependency used in the webapp and on the login page served by the API.
Not sure if this should also be a dependency of `api` and point the symlink there. If we want to do that, this won't be a big change.